study-builder/studies/osteo/sendgrid_emails.conf

```diff
@@ -0,0 +1,263 @@
+{
+  "sendgridEmails": [
+    {
+      "templateId": ${emails.exitRequest},
+      "name": "OSTEO_EXIT_REQUEST",
+      "subject": "Participant Withdraw Request",
+      "filepath": "emails/exit_request.html",
+      "render": true
+    },
```

Comment:
Is there a better way to configure this? I think one issue is there's no clear "identifier" for the email templates. We need to create the emails first before we can put those template ids into vault and thus use them in things like `${email.exitRequest}` substitutions. So using template id doesn't seem like the way to go. Perhaps we can use the `name`? But name in SendGrid is not unique...

Reply:
I ended up using the email's "key" as the identifier here instead of reference the template id directly.